go.mod

```diff
@@ -4,5 +4,5 @@ go 1.14
 
 require (
 	github.com/hashicorp/terraform v0.12.28
-	github.com/shakenfist/client-go v0.2.0
+	github.com/shakenfist/client-go v0.2.2
```

Comment:
We need to have a think about when this is changed.
Simple sync with SF or only when client-go has required changes for TF.
(I'll merge my waiting PR after this one, so that my PR bumps it 0.2.3 - which is required)

Reply:
I don't think I intended to bump this, go build did it for me maybe?